seqr/views/apis/igv_api.py

```diff
@@ -2,16 +2,22 @@
 import json
 import re
 import requests
+import time
+
 from django.http import StreamingHttpResponse, HttpResponse
 
 from seqr.models import Individual, IgvSample
-from seqr.utils.file_utils import file_iter, does_file_exist
+from seqr.utils.file_utils import file_iter, does_file_exist, run_command, is_google_bucket_file_path
 from seqr.views.utils.file_utils import save_uploaded_file
 from seqr.views.utils.json_to_orm_utils import get_or_create_model_from_json
 from seqr.views.utils.json_utils import create_json_response
 from seqr.views.utils.orm_to_json_utils import  get_json_for_sample
 from seqr.views.utils.permissions_utils import get_project_and_check_permissions, check_project_permissions, \
     login_and_policies_required, pm_or_data_manager_required
+from seqr.utils.redis_utils import safe_redis_get_json, safe_redis_set_json
+
+EXPIRATION_TIME_IN_SECONDS = 3600 - 5
+IGV_CACHE_KEY = 'seqr-igv-redis_cache_key'
```

Comment:
Its a bit odd to put the words "cache_key" in a cache key. This is also an incredibly confusing variable name - it sounds like a prefix for caching igv data, it would be much clearer if you specify that this is caching the access token (which isn't even IGV specific). I would name both the variable and the key something like `GS_STORAGE_ACCESS_TOKEN`

Reply:
I did it as you suggest. But the Codacy keeps reporting 'possible hardcoded credential'. I agree that it's not for IGV only. Maybe I can use `GS_STORAGE_SECRET_CACHE_KEY` for the name of the constant and `gs_storage_secrect_cache_entry` for the cache key itself. I hope the Codacy won't report any error.

Reply:
using bad variable names to get around codacy is a bad idea. putting the word "secret" in is even worse because a secret is a specific kubernetes thing in our deployment and this is not a secret. We could try adding "temp" in to indicate that this roatates. Does it complain about 
```
GS_STORAGE_TEMP_ACCESS_KEY = 'GS_STORAGE_TEMP_ACCESS_KEY'
```

Reply:
I use `GS_STORAGE_ACCESS_CACHE_KEY` now. Codacy doesn't complain. Do you think it is okay?

Reply:
I think its fine. Also a thing I forgot, if you put the comment `# nosec` at the end fo the line codacy won't complain, so in the future we can also do that